content/docs/ops/security/harden-docker-images/index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Harden Docker Container Images
+description: Properly securing Docker containers is a complicated and time consuming task.
+weight: 80
+aliases:
+    - /help/ops/security/harden-docker-images
+---
+To ease the process of hardening docker images, Istio provides a set of images based on  [*distroless images* provided by Google](https://github.com/GoogleContainerTools/distroless)
```

Comment:
I'd remove `provided by Google` as distroless is not an official Google project and we want Istio to be as independent as possible.  I'd also remove the bolding of distorless images.